File: 20140207/fizzbuzz.hs
module Main where

toStr :: Int -> String
toStr n =
      if (n `mod` 15) == 0 then "FizzBuzz"
      else if (n `mod` 3) == 0 then "Fizz"
      else if (n `mod` 5) == 0 then "Buzz"
      else show n

fizzbuzz :: [Int] -> [String]
fizzbuzz a = map (toStr) a

main :: IO ()
main  = do
     print(fizzbuzz([1..100]))

File: 20140217/fizzbuzz.py
def to_str(num):
    if num % 15 == 0:
        return "FizzBuzz"
    elif num % 3 == 0:
        return "Fizz"
    elif num % 5 == 0:
        return "Buzz"
    else:
        return str(num)

def fizzbuzz(array):
    for i in array:
        print(to_str(i))

fizzbuzz(range(1,100))

